universal-application-tool-0.0.1/app/services/applicant/ApplicantData.java

```diff
@@ -62,4 +64,18 @@ public void setCreatedTime(Instant i) {
   public String asJsonString() {
     return this.jsonData.jsonString();
   }
+
+  @Override
+  public boolean equals(@Nullable Object object) {
+    if (object instanceof ApplicantData) {
+      ApplicantData that = (ApplicantData) object;
+      return Objects.equals(this.jsonData.jsonString(), that.jsonData.jsonString());
```

Comment:
suggestion (blocking): Add a comment explaining why we can't use DocumentContext here

Reply:
Done